src/main/java/com/solace/samples/TopicPublisher.java

```diff
@@ -26,57 +30,74 @@
 import javax.jms.Session;
 import javax.jms.TextMessage;
 import javax.jms.Topic;
+
 import com.solacesystems.jms.SolConnectionFactory;
 import com.solacesystems.jms.SolJmsUtility;
 
+/**
+ * Publishes a messages to a topic using Solace JMS 1.1 API implementation.
+ * 
+ * This is the Publisher in the Publish/Subscribe messaging pattern.
+ */
 public class TopicPublisher {
 
-    public void run(String... args) throws Exception {
+    final String SOLACE_VPN = "default";
```

Comment:
Mark, how do want it to be handled? I copied the existing argument code as is. Please be specific.